R/mod_packageDependencies.R

```diff
@@ -160,11 +162,13 @@ packageDependenciesServer <- function(id, selected_pkg, user, changes, parent) {
         showHelperMessage()
       } else {
         req(depends())
+        # Create metric grid card.
+        metricGridServer(id = 'metricGrid', metrics = cards) 
```

Comment:
Just noticed that the server portion of the module is embedded in a `renderUI()` resulting in multiple calls.

Reply:
I had incorrectly assumed we would not have a problem with suggests.
Looks like I will need to do a tryCatch() like I did with depends.

And I will take a look at moving the metricGridServer() function

Reply:
I saw your issue in the `{riskmetric}` package.
package `{abc}` only has `Depends` and so `riskmetric::assess_dependencies()` fails